OpenRA.Mods.Common/UpdateRules/UpdatePath.cs

```diff
@@ -115,7 +115,7 @@ public class UpdatePath
 				new RemoveAttackSuicides(),
 			}),
 
-			new UpdatePath("release-20190314", "playtest-20190825", new UpdateRule[]
+			new UpdatePath("release-20190314", "release-20191117", new UpdateRule[]
 			{
 				// Prep only changes here
```

Comment:
I suppose we can remove this comment now (or introduce a new update path for prep, which I don't think we'll need however).